devito/ops/utils.py

```diff
@@ -5,7 +5,6 @@
 namespace = OrderedDict()
 
 # OPS Kernel strings.
-namespace['ops_acc'] = 'OPS_ACC'
```

Comment:
can we not drop this yet?

Reply:
why?

Reply:
if we'll cover the old syntax as well, we will have to add it again 
anyways, i understand it's currently not being used. 
so maybe it's ok to add it again when we push new PR

Reply:
why would you want to support the old syntax if the new one is just better? 
having to support two things increases the maintenance burden. Would it be that difficult to update whatever you have to the new syntax?